packages/internal-lib-build/package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
 		"": {
 			"name": "internal-lib-build",
-			"version": "2.9.0",
+			"version": "2.9.1",
```

Comment:
Other packages have their package lock updated too, but it's simply syncing them to the current package version.